x-pack/plugins/enterprise_search/public/applications/enterprise_search_relevance/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues } from '../__mocks__/kea_logic';
+import '../__mocks__/shallow_useeffect.mock';
+import '../__mocks__/enterprise_search_url.mock';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
```

Comment:
I tried to convert 3 enzyme test to react-testing-library but encountered difficulties with the tests related to EnterpriseSearchRelevance. I have created a jira ticket so that I can spend some time later to convert them to react-testing-library. Except for 3 enzyme tests, all the tests have been written using react-testing-library.